plugins/woocommerce/client/blocks/assets/js/blocks/product-specifications/types.ts

```diff
@@ -7,4 +7,12 @@ type Context = {
 	context: { postId: string; postType: string };
 };
 
-export type ProductSpecificationsEditProps = BlockEditProps< object > & Context;
+interface ProductSpecificationsAttributes {
+	fontSize?: string;
```

Comment:
I don't think `fontSize` should be listed as an attribute. I see it's mentioned in the `block.json` but it's coming from `useBlockProps` thanks to `supports` declaration. And that means it's not part of `ProductSpecificationsEditProps`.

Reply:
Yep, good catch, we should remove `fontSize`.